internal/rbac/gen/yamldata.go

```diff
@@ -126,11 +126,15 @@ func loadSchema(filename string) (*schema, error) {
 
 func generateTSConstants(output io.Writer, permissions []permissionNamespace) {
 	fmt.Fprintln(output, TSGeneratedByTarget)
-	for _, permission := range permissions {
+	var permissionNames = make([]string, len(permissions))
```

Comment:
nit: use `:=` for combo declaration/assignment

```suggestion
	permissionNames := make([]string, len(permissions))
```